common/iterator/Iterators.java

```diff
@@ -50,6 +50,11 @@
         return new BaseIterator<>(Either.second(collection.iterator()));
     }
 
+    @SafeVarargs
+    public static <T> FunctionalIterator<T> iterate(T... elements) {
+        return new BaseIterator<T>(Either.second(list(elements).iterator()));
```

Comment:
This should call the pre-existing method above it. And move this method above it.